backend/core/settings.py

```diff
@@ -43,15 +43,16 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "rest_framework",
+    "ctgov",
 ]
 
-REST_FRAMEWORK = {
-    # Use Django's standard `django.contrib.auth` permissions,
-    # or allow read-only access for unauthenticated users.
-    "DEFAULT_PERMISSION_CLASSES": [
-        "rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly"
-    ]
-}
+# REST_FRAMEWORK = {
```

Comment:
Can you remove this? It's better to not leave commented out code in the code base.